arches/install/arches-templates/project_name/settings.py-tpl

```diff
@@ -124,6 +117,8 @@ INSTALLED_APPS = (
     "{{ project_name }}",
 )
 
+ARCHES_APPLICATIONS = ()
```

Comment:
> We should also update the settings template to fix INSTALLED_APPS, but that's probably out of scope for this PR.

Agreed, but both should happen simultaneously. Since one is out of scope I'd say file a ticket and punt

Reply:
👍 #9942